src/libraries/System.Private.CoreLib/src/System/Buffers/Text/FormattingHelpers.CountDigits.cs

```diff
@@ -52,60 +52,83 @@ public static int CountDigits(UInt128 value)
             return digits;
         }
 
+        // Based on do_count_digits from https://github.com/fmtlib/fmt/blob/662adf4f33346ba9aba8b072194e319869ede54a/include/fmt/format.h#L1124
+        /*
+          Formatting library for C++
```

Comment:
This should be also included in https://github.com/dotnet/runtime/blob/main/THIRD-PARTY-NOTICES.TXT .